endoreg_db/models/other/material.py

```
@@ -1,4 +1,11 @@
 from django.db import models
+from typing import TYPE_CHECKING
+
+from endoreg_db.models.administration.product import product_material
```

Ruff (0.8.2)

`endoreg_db.models.administration.product.product_material` imported but unused

Remove unused import: `endoreg_db.models.administration.product.product_material`

(F401)